tests/playwright/config/global-setup.js

```diff
@@ -7,8 +7,9 @@ module.exports = async ( config ) => {
 	await page.goto( `${ config.projects[ 0 ].use.baseURL }/wp-admin` );
 
 	await page.waitForSelector( 'text=Log In' );
-	await page.fill( 'input[name="log"]', 'admin' );
-	await page.fill( 'input[name="pwd"]', 'password' );
+	await page.fill( 'input[name="log"]', process.env.UNAME || 'admin' );
+	await page.waitForTimeout( 500 );
+	await page.fill( 'input[name="pwd"]', process.env.PASSWD || 'password' );
```

Comment:
It should be tested in a clean environment.

Reply:
Do clean environments necessarily have the same credentials?